tests/test_comparercollection.py

```diff
@@ -291,6 +291,63 @@ def test_filter_by_attrs_custom(cc):
     assert cc3[0] == cc[0]
 
 
+def test_skill_by_attrs_gtype(cc):
+    sk = cc.skill(by="attrs:gtype")
+    assert len(sk) == 2
+    assert sk.data.index[0] == "point"
```

Comment:
It seems overly specific to assert that the index is sorted in this order.

Isn't it enough to verify that:
```python
assert "point" in sk.index
assert "track" in sk.index
```

Reply:
true - will fix in next PR which will be on sorting